app/modules/relationships/models.py

```diff
@@ -0,0 +1,116 @@
+# -*- coding: utf-8 -*-
+"""
+Relationships database models
+--------------------
+"""
+
+from sqlalchemy_utils import Timestamp
+
+from app.extensions import HoustonModel, db
+from datetime import datetime  # NOQA
+import app.extensions.logging as AuditLog
+
+import uuid
+
+import logging
+
+log = logging.getLogger(__name__)
+
+
+class RelationshipIndividualMember(db.Model, HoustonModel):
+
+    guid = db.Column(
+        db.GUID, default=uuid.uuid4, primary_key=True
+    )  # pylint: disable=invalid-name
+
+    relationship_guid = db.Column(db.GUID, db.ForeignKey('relationship.guid'))
+    relationship = db.relationship(
+        'Relationship', backref=db.backref('individual_members')
+    )
+
+    individual_guid = db.Column(db.GUID, db.ForeignKey('individual.guid'))
+    individual = db.relationship('Individual', backref=db.backref('relationships'))
+
+    individual_role = db.Column(db.String, nullable=False)
+
+    def __init__(self, individual, individual_role, **kwargs):
+        self.individual = individual
+        self.individual_role = individual_role
+        self.individual_guid = individual.guid
+
+
+class Relationship(db.Model, Timestamp):
+    """
+    Relationships database model.
+    """
+
+    guid = db.Column(
+        db.GUID, default=uuid.uuid4, primary_key=True
+    )  # pylint: disable=invalid-name
+
+    start_date = db.Column(
+        db.DateTime, index=True, default=datetime.utcnow, nullable=True
+    )
+    end_date = db.Column(db.DateTime, index=True, default=datetime.utcnow, nullable=True)
+
+    type = db.Column(db.String, nullable=True)
+
+    def __init__(
+        self,
+        individual_1_guid,
+        individual_2_guid,
+        individual_1_role,
+        individual_2_role,
+        **kwargs
+    ):
+        if (
+            individual_1_guid
+            and individual_2_guid
+            and individual_1_role
+            and individual_2_role
+        ):
+
+            from app.modules.individuals.models import Individual
+
+            individual_1 = Individual.query.get(individual_1_guid)
```

Comment:
What happens if either of these return None?

Reply:
Good point, these should be validated. Will do that. I favor guids over the object, as it seems we are more likely to have a guid from an API call than a representation of the object in most cases, and if we can avoid the database hit we should.